src/i18n/messages/nl.json

```
@@ -20,7 +20,7 @@
     "originalDefault": "Log out"
   },
   "1+Q55M": {
-    "defaultMessage": "You are verifying the email address <link>{emailAddress}</link>. First, we need to send you a verification code on this email address. Then, you need to enter the code to confirm it.",
+    "defaultMessage": "Om door te gaan moet je jouw e-mailadres <link>{emailAddress}</link> bevestigen. We sturen je eerst een bevestigingscode naar dit adres. Vervolgens moet je deze code ter bevestiging invoeren.",
```

Zin liep niet lekker en consistentie.

```suggestion
    "defaultMessage": "Om door te gaan moet je jouw e-mailadres <link>{emailAddress}</link> bevestigen. We sturen eerst een bevestigingscode naar dit e-mailadres. Vervolgens moet je deze bevestigingscode hier invoeren.",
```